downloads/swan-lake-release-notes/swan-lake-2201.4.2/RELEASE_NOTE.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: ballerina-left-nav-release-notes
+title: 2201.4.2 (Swan Lake) 
+permalink: /downloads/swan-lake-release-notes/2201-4-2/
+active: 2201-4-2
+redirect_from: 
+    - /downloads/swan-lake-release-notes/2201-4-2
+    - /downloads/swan-lake-release-notes/2201.4-2/
+    - /downloads/swan-lake-release-notes/2201-4-2-swan-lake/
+    - /downloads/swan-lake-release-notes/2201-4-2-swan-lake
+---
+
+## Overview of Ballerina Swan Lake 2201.4.2
+
+<em>Swan Lake 2201.4.2 is the second patch release of Ballerina 2201.4.0 (Swan Lake Update 4) and it includes a new set of improvements to the language and developer tooling.</em>
+
+## Update Ballerina
+
+**If you are already using Ballerina 2201.0.0 (Swan Lake)**, run either of the commands below to directly update to 2201.4.2 using the [Ballerina Update Tool](/learn/cli-documentation/update-tool/).
+
+`bal dist update` (or `bal dist pull 2201.4.2`)
+
+**If you are using a version below 2201.0.0 (Swan Lake)**, run the commands below to update to 2201.4.2.
+
+1. Run `bal update` to get the latest version of the Update Tool.
+
+2. Run `bal dist update` ( or `bal dist pull 2201.4.2`) to update your Ballerina version to 2201.4.2.
+
+However, if you are using a version below 2201.0.0 (Swan Lake) and if you already ran `bal dist update` (or `bal dist pull 2201.4.2`) before `bal update`, see [Troubleshooting](/downloads/swan-lake-release-notes/swan-lake-2201.0.0#troubleshooting) to recover your installation.
+
+## Install Ballerina
+
+If you have not installed Ballerina, then, download the [installers](/downloads/#swanlake) to install.
+
+## Language updates
+
+### Bug fixes
+
+To view bug fixes, see the [GitHub milestone for 2201.4.2 (Swan Lake)](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.4.2+is%3Aclosed+label%3AType%2FBug).
+
+## Developer tools updates
+
+### Bug fixes
+
+To view bug fixes, see the GitHub milestone for 2201.4.2 (Swan Lake) of the repositories below.
+
+- [OpenAPI](https://github.com/ballerina-platform/openapi-tools/issues?q=is%3Aissue+milestone%3A%22Swan+Lake+2201.4.2%22+is%3Aclosed+)
+
+## Standard library updates
+
+### Bug fixes
+
+To view bug fixes, see the [GitHub milestone for 2201.4.2 (Swan Lake)](https://github.com/ballerina-platform/ballerina-standard-library/milestone/28?closed=1).
```

Comment:
```suggestion
To view bug fixes, see the [GitHub milestone for 2201.4.2 (Swan Lake)](https://github.com/ballerina-platform/ballerina-standard-library/milestone/28?closed=1).
```
Can we be consistent with the others and get a milestone link like below for standard library also? We have done this correctly in the [2201.3.3 release note](https://ballerina.io/downloads/swan-lake-release-notes/swan-lake-2201.3.3#standard-library-updates).

`https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aissue+milestone%3A2201.3.3+label%3AType%2FBug+is%3Aclosed+`

Reply:
updated in [7aa8117](https://github.com/ballerina-platform/ballerina-dev-website/pull/6780/commits/7aa81171ac6200b2d7d80eb3ce51242f28c64d66)